src/test/java/com/final_assignment/UserMapperTest.java

```diff
@@ -79,14 +78,39 @@ class UserMapperTest {
     }
 
     @Test
-    void insert() {
+    @Sql(
+        scripts = {"classpath:/sqlannotation/delete-users.sql", "classpath:/sqlannotation/insert-users.sql"},
+        executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD
+    )
+    @Transactional
+    void 新規登録ができること() {
+        User newUser = new User("soya", "soya@example.com");
+        userMapper.insert(newUser);
+        List<User> users = userMapper.findAll();
+        assertEquals(4, users.size());
+        assertTrue(users.stream().anyMatch(user -> "soya".equals(user.getName()) && "soya@example.com".equals(user.getEmail())));
```

Comment:
```suggestion
        assertTrue(users.stream().anyMatch(user -> user.getName().equalst("soya") && user.getEmail.equals("soya"));
```
個人的にはヨーダ記法じゃないほうがいいかなと思いました。